dotnet-build.sh

```diff
@@ -26,7 +26,7 @@ if [ -n "$MYGET_ACCESS_TOKEN" ]; then
   fi
 fi
 
-CS_PROJECT_FILE="Api/Api.csproj"
+CS_PROJECT_FILE="src/Api/Api.csproj"
```

Comment:
This should be a **default** variable but right now it's ALWAYS overwritten.
Please use:

`CS_PROJECT_FILE="${CS_PROJECT_FILE:-"src/Api/Api.csproji"}"`

This will allow you to overwrite the CS_PROJECT_FILE from outside of the container